bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/internal/SecureHttpClientFactory.java

```diff
@@ -174,7 +174,7 @@ public Void run() {
                 if (cause instanceof RuntimeException) {
                     throw (RuntimeException) cause;
                 } else {
-                    throw new RuntimeException(cause);
```

Comment:
With the lines above this looks like the RuntimeException is thrown on purpose here. @sobimibos can you clarify here what we can do alternatively? Thanks.

Reply:
I discarded the changes and added an exception for this class for now.